packages/firestore/src/local/encoded_resource_path.ts

```diff
@@ -72,7 +72,7 @@ const encodedEscape = '\u0011';
 /**
  * Encodes a resource path into a IndexedDb-compatible string form.
  */
-export function encode(path: ResourcePath): EncodedResourcePath {
+export function encodeResourcePath(path: ResourcePath): EncodedResourcePath {
```

Comment:
What is your rationale for renaming these functions?  It's not that I disagree, it's just that the motivation is unclear to me.

Reply:
The are now used as top-level functions. I renamed them to be more descriptive than just "encode/decode".